src/widget/loginscreen.cpp

```diff
@@ -40,7 +40,7 @@ LoginScreen::LoginScreen(QWidget *parent) :
     ui->setupUi(this);
 
     // permanently disables maximize button https://github.com/tux3/qTox/issues/1973
-    this->setWindowFlags(windowFlags() &! Qt::WindowMaximizeButtonHint);
+    this->setWindowFlags(windowFlags() &~ Qt::WindowMaximizeButtonHint);
```

Comment:
This was actually right before. An XOR would enable the `Qt::WindowMaximizeButtonHint` flag and thus declines the description above. Please revert